doc/user/content/ingest-data/network-security/static-ips.md

```diff
@@ -31,7 +31,24 @@ connections from your region's egress IP addresses. You must allow connections
 from all egress IP addresses associated with your region. Connections may
 originate from any one of the egress IP addresses.
 
-To find the egress IP addresses associated with your region, you can query the
+## AWS Static Ips
+
+Each region of materialize uses a static set of CIDRs for egress IPs.
+
+Region      | CIDR
+------------|------------
+`us-east-1` | 1.2.3.0/28
```

Comment:
should we put our actual IPs here?

Reply:
yeah, I wanted to see what we thought of this before before putting our actual IPs/CIDRs here.